lib/stripe/file_upload.rb

```diff
@@ -18,5 +18,10 @@ def self.all(filters={}, opts={})
       response, opts = request(:get, url, filters, opts)
       Util.convert_to_stripe_object(response, opts)
     end
+
+    def self.retrieve(id, opts={})
```

Comment:
`opts` can actually be a string, so it's not safe to call merge in this case. It looks like this entire file suffers from this problem. We'll want to add the following to `all`, `create`, and `retrieve`.

```ruby
opts = Util.normalize_opts(opts)
```